app/services/onlinetesting/OnlineTestService.scala

```diff
@@ -253,12 +259,22 @@ trait OnlineTestService {
   }
 
   private[services] def buildInviteApplication(application: OnlineTestApplication, token: String, userId: Int, scheduleId: Int) = {
-    val onlineTestCompletedUrl = gatewayConfig.candidateAppUrl + "/fset-fast-stream/online-tests/complete/" + token
+    val scheduleCompletionUrl = gatewayConfig.candidateAppUrl + "/fset-fast-stream/online-tests/complete/" + token
```

Comment:
I don't think so.  This endpoint is just concerned with completing a test using a token as a key.  If we had endpoints for each test the implementation for them would be mostly the same.